contracts/VestingEscrowSimple.vy

```diff
@@ -133,7 +126,7 @@ def locked() -> uint256:
 
 
 @external
-def claim(beneficiary: address = msg.sender, amount: uint256 = MAX_UINT256):
+def claim(beneficiary: address = msg.sender, amount: uint256 = max_value(uint256)):
```

Comment:
changes for release notes:
- add tether support
- anyone could claim on behalf or recipient

the latter should probably be a flag that you enable